Packs/Legacy/Integrations/integration-Pwned.yml

```diff
@@ -345,4 +345,4 @@ script:
   runonce: false
 deprecated: true
 tests:
-  - Pwned test
+  - No tests
```

Comment:
why did you change this?

Reply:
The test is not registered in conf.json and the integration is deprecated